docs/administrationGuide.md

```diff
@@ -59,28 +59,29 @@ service iotagent-lwm2m stop
 
-# <a name="configuration"> Configuration </a>
-There are two ways to provide the IOT Agent with a configuration set: passing the name of a config file (related to the 
-root folder of the project) or customise the example `config.js` in the root. 
+# Configuration
+There are two ways to provide the IOT Agent with a configuration set: passing the name of a config file (related to the
+root folder of the project) or customise the example `config.js` in the root.
 
-The configuration file is divided in two sections: one standard section for the NGSI North bound `ngsi`, and another 
-one for the specific Lightweight M2M South bound, `lwm2m`. The former follows the same format described for the Node.js 
-IOT Agent Framework, described [here](https://github.com/telefonicaid/iotagent-node-lib#global-configuration).
+The configuration file is divided in two sections: one standard section for the NGSI traffic North of the IoT Agent `ngsi`, and another
+one for the specific Lightweight M2M traffic South of the IoT Agent, `lwm2m`. The former follows the same format described for the Node.js
+IoT Agent Framework, described [here](https://github.com/telefonicaid/iotagent-node-lib#global-configuration).
 
-The latter configures the Lightweight M2M library used for communicating with the devices, as described 
+The latter configures the Lightweight M2M library used for communicating with the devices, as described
 [here](https://github.com/telefonicaid/lwm2m-node-lib#-configuration) (`server` section).
 
 These are the specific LWM2M parameters that can be configured for the agent:
 
 - **logLevel**: level of logs for the IOTAgent specific information. E.g.: 'DEBUG'.
 - **port**: UDP port where the IOTAgent will be listening. E.g.: 60001.
-- **delayedObservationTimeout**: When a LWM2M client has active attributes, the IOTA sends an observe instruction for 
-each one, just after the client registers. This may cause cause an error when the client takes too long to start listening, 
-as the observe requests may not reach its destiny. This timeout (ms) is used to give the client the opportunity to create 
+- **delayedObservationTimeout**: When a LWM2M client has active attributes, the IOTA sends an observe instruction for
+each one, just after the client registers. This may cause cause an error when the client takes too long to start listening,
+as the observe requests may not reach its destiny. This timeout (ms) is used to give the client the opportunity to create
 the listener before the server sends the requests.
 - **defaultType**: for the cases when no type can be assigned to a device (no pre-provision or path assignation of type), this type will be assigned by default. E.g.: 'Device'
 - **types**: for IOTAgents with multiple southbound paths, this attribute maps attribute types (defined either in the configuration file or by using the Device Configuration API) to southbound interfaces. E.g.:
-```
+
+```json
```

Comment:
It seems that after this change, markdown renders

![imagen](https://user-images.githubusercontent.com/1534240/42824207-a20a8318-89df-11e8-915d-bbad25e32447.png)

while in master it is rendered correctly:

![imagen](https://user-images.githubusercontent.com/1534240/42824235-b0a939b4-89df-11e8-9fed-babf007b3a92.png)

This should be fixed...

Reply:
Maybe it is a syntax problem... Maybe this fragment should use `javascript` syntax?

Reply:
I'll just remove the syntax higher on this config for now
fixed d1cf01f